Labs/Alyssa_Giedd_Lab2.ipynb

```diff
@@ -0,0 +1,477 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x22711e22d68>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#C) Show how the probability distribution evolves as you average days. Calculate for many different ranges of days, \n",
+    "#and explore the shape of the distribution as the number of days becomes larger. Discuss this in relation to both \n",
+    "#B) and the central limit theoem.\n",
+    "days = [12, 15, 18, 21]\n",
+    "for index in range(len(days)):\n",
+    "    #Sums to plot\n",
+    "    bg_summation = convolve_multiple(y_avg, days[index])\n",
+    "    #Produce graph with the specified days worth of data\n",
+    "    plt.step(np.linspace(0, len(bg_summation), len(bg_summation))/days[index], bg_summation, label = str(days[index]) + \" day sum\")\n",
+    "\n",
+    "plt.legend()\n",
+    "plt.title(\"Average Background for a Range of Days\")\n",
```

Comment:
also here, discuss the distribution shapes and the central limit theorem in relation to the range of days that you averaged  over